auth/LoginHandler.php

```diff
@@ -77,7 +76,7 @@ public function logout() {
 
 		$this->persistentCookie->invalidate_cookie();
 		Auth::checkout();
-		Session::delete('authUser');
+		Session::delete();
```

Comment:
think I meant to do a Session::unset('authUser'); here

```suggestion
		Session::unset('authUser');
```